test/plugins/cleanupIds.24.svg

```diff
@@ -0,0 +1,37 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24">
```

Comment:
Could we keep the tests more concise?

Most of the SVG is unnecessary to reproduce the issue. It just keeps the test more focused so it's easier to see what's tested, and reduces overlap with other tests when making changes.

When tests contain more than they need, they may need updating due to changes unrelated to the purpose of the test. Especially without a description, it's unclear if the purpose of the test is being undermined. It's one of the reasons I'm reluctant to change old tests for now.

I think something like this is better, it describes the problem, renders well for a visual comparison, and easy to review what's supposed to change without as much noise:

```svg
Correctly pull IDs from begin attribute even if there are semi-colon seperated
values.

See: https://github.com/svg/svgo/pull/1775

===

<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
  <circle cx="12" cy="12">
    <animate id="owo" fill="freeze" attributeName="r" begin="0;uwu.end" dur="1.2s" values="0;11"/>
  </circle>
  <circle cx="12" cy="12">
    <animate id="uwu" fill="freeze" attributeName="r" begin="owo.begin+0.2s" dur="1.2s" values="0;11"/>
  </circle>
</svg>

@@@

<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
    <circle cx="12" cy="12">
        <animate id="b" fill="freeze" attributeName="r" begin="0;a.end" dur="1.2s" values="0;11"/>
    </circle>
    <circle cx="12" cy="12">
        <animate id="a" fill="freeze" attributeName="r" begin="b.begin+0.2s" dur="1.2s" values="0;11"/>
    </circle>
</svg>
```